### PR TITLE
Branch tsima

### PR DESCRIPTION
**Pull request by tsimamada**

### CODE REVIEW

**Comment by tsimamada** [comment]
https://rawgit.com/tsimamada/tsimamada.github.io/branch_tsima/#!pages/profiles/tsimamada.md

**Comment by bsocial** [comment]
This is a good start. Suggest that you look at some of the other examples from the vi team.

File: pages/profiles/tsimamada.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+* #### My name is Rasolo niaina
+* I'm from **Madagascar**
+* _I work at_ [Minister of National Education][site men]
```
Comment (xyb994) [inline pages/profiles/tsimamada.md]: Seems like this link would not work without adding `http://` to the url on line 5

**Comment by kylemathias** [comment]
I would add a little more. Try looking at [team.md](http://open-learning-exchange.github.io/#!pages/team.md) for some examples.

**Pull request merged**